src/main/java/core/basesyntax/FootballMatch.java

```diff
@@ -17,6 +14,15 @@
      *         если не угадал ничего - возвращать 0.
      */
     public int matchResult(int result1, int result2, int bet1, int bet2) {
-        return 0;
+        int dif1 = result1 - result2;
```

Comment:
try to simplify
we can implement this method without using redundant variables and method